mne/cov.py

```diff
@@ -1431,11 +1435,12 @@ def _get_whitener_data(info, noise_cov, picks, diag=False, rank=None,
         noise_cov = cp.deepcopy(noise_cov)
         noise_cov['data'] = np.diag(np.diag(noise_cov['data']))
 
-    scalings_ = dict(mag=1e12, grad=1e11, eeg=1e5)
+    _scalings = DEFAULTS['scalings']
     if isinstance(scalings, dict):
-        scalings_.update(scalings)
```

Comment:
is there a reason for this syntax over:

    if scalings is None:
        scalings = DEFAULTS['scalings']

?